drivers/cc110x/cc110x-netdev2.c

```diff
@@ -84,6 +84,8 @@ static inline int _get_iid(netdev2_t *netdev, eui64_t *value, size_t max_len)
     cpuid_get(cpuid);
 
     memcpy(eui64 + 8 - n, cpuid, n);
+    eui64[sizeof(eui64_t) - 2] = 0;
+    eui64[sizeof(eui64_t) - 1] = cpuid[CPUID_LEN-1];
```

Comment:
Yes, but the cc110x does filtering based on cpuid[CPUID_LEN-1]. So if we set the last 16 bits to this value, the node accepts the packet properly and replies to ping packets.